deps.ts

```diff
@@ -1,4 +1,4 @@
-export * from "https://deno.land/x/web_bson@v0.2.2/mod.ts";
```

Comment:
don't remove it again
Bson is not standard in the nodejs driver

Reply:
is there a reason why there is bson.ts and not deps.?

Reply:
this deps.ts file is for internal usages only
I already exported all bson types in mod.ts

Reply:
but then just reexport it from bson.ts

instead of changing every path

Reply:
This is not finished.
I'm going to only use `Bson.*` for internal usages. It's more readable

Reply:
Yeah only internal but still
Why would `new Bson.ObjectId()` be more readable? IMO it just introduces noise as the reader doesn't need to know an ObjectId is from Bson everytime they create a new objectid

Reply:
imagine having a function like this
```ts
function(): Timestamp {
}
```

You should check the `Timestamp` type to see what actually it is. (because Timestamp naming is too generic)

```ts
function(): Bson.Timestamp {
}
```
but here you are pretty sure that's a bson type

Reply:
This really is not so important.
Okay, I'll not touch it

Reply:
But this is internal mongo driver code

I don't see where we actually have a problem with overlapping of types